packages/flutter/lib/src/widgets/_web_selectable_region_context_menu_web.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+
+import 'package:flutter/rendering.dart';
+
+import '_dart_ui_io.dart' if(dart.library.html) 'dart:ui' as ui;
```

Comment:
I don't think you need the conditional import or `_dart_ui_io.dart`. Instead, you can add an `// ignore` to ignore the analyzer warning about `ui.platformViewRegistry`.

Reply:
if i remove this line, it is a compiling error. I don't think the ignore can fix this?

Reply:
nvm i think it worked